src/graphapi/graphapi.cpp

```diff
@@ -78,6 +79,10 @@ miopenBackendCreateDescriptor(miopenBackendDescriptorType_t descriptorType,
         case MIOPEN_BACKEND_VARIANT_PACK_DESCRIPTOR:
             outputDesciptor = new miopen::graphapi::BackendVariantPackDescriptor(); break;
 
+        case MIOPEN_BACKEND_MATMUL_DESCRIPTOR:
+            outputDesciptor = new miopen::graphapi::BackendMatmulDescriptor();
+            break;
+
```

Comment:
Place the case in alphabet order (to reduce the probability of future merge conflicts)